src/icons/raw/Heart.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In relation to the comment I wrote in the `stories/index.tsx` file,
the new `svg` files should be added to the `assets/icons` directory.